src/components/AddEventToPublishedScheduleForm/AddEventToPublishedScheduleForm.jsx

```diff
@@ -104,7 +106,14 @@ const AddEventToPublishedScheduleForm = ({ closeForm }) => {
     } else {
       setPlannedEvents(plannedEvents.filter(e => e.id != -1));
     }
-  }, [formData])
+  }, [formData]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'checkboxVal', 'eventData.id', 'plannedEvents', and 'setPlannedEvents'. Either include them or remove the dependency array.